AndroidNativeKotlinTemplate/app/src/main/java/com/salesforce/androidnativekotlintemplate/MainActivity.kt

```diff
@@ -156,4 +163,16 @@ class MainActivity : SalesforceActivity() {
             }
         })
     }
-}
\ No newline at end of file
+
+    // region QR Code Login Via Salesforce Identity API UI Bridge Public Implementation
+
+    private fun useQrCodeLoginUrl() {
+        val uiBridgeApiParameters = uiBridgeApiParametersFromQrCodeLoginUrl(intent.data.toString())
+        SalesforceSDKManager.getInstance().apply {
```

Comment:
Here's the app's opportunity to provide the front door URL before it soon gets the REST client, which will automatically display the login activity as a side effect.

Are there any other paths to displaying the login activity we might need to intercept? Once these parameters are set, the login activity will use them one time.

Reply:
>Are there any other paths to displaying the login activity we might need to intercept? Once these parameters are set, the login activity will use them one time.

I don't think so.